android/sdl_android/src/main/java/com/smartdevicelink/managers/lockscreen/LockScreenManager.java

```diff
@@ -99,6 +97,14 @@ public LockScreenManager(LockScreenConfig lockScreenConfig, Context context, ISd
 		customView = lockScreenConfig.getCustomView();
 		lockScreenEnabled = lockScreenConfig.isEnabled();
 		deviceLogoEnabled = lockScreenConfig.isDeviceLogoEnabled();
+		displayMode = lockScreenConfig.getDisplayMode();
```

Comment:
When the display mode is set to `DISPLAY_MODE_ALWAYS`, the lock screen cannot be dismissed even if `enableDismissGesture=true`. not sure if that is intentional or a bug.